### ISSUE TITLE
Missing word in Deferred intro doc

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6403]
|<img alt="terrycojones's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| terrycojones reported|
|-|-|
|Trac ID|trac#6403|
|Type|enhancement|
|Created|2013-04-04 02:20:08Z|

http://twistedmatrix.com/documents/current/core/howto/defer-intro.html contains (towards the bottom) "we can see what would happen when we them around." which is missing a word (swap?).

Attachments:

* [6403.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/58d/58d49df0c848862006408092d9f4c0c874dba4fe/0ce4bc905be24d66b19c10cbb4a71b8d6d50d60f.patch) (539 bytes) - added by kkpattern on 2013-04-25 12:49:49Z - Add the word "swap".

<details><summary>Searchable metadata</summary>

```
trac-id__6403 6403
type__enhancement enhancement
reporter__terrycojones terrycojones
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__deferred_easy_documentation deferred easy documentation
time__1365042008000000 1365042008000000
changetime__1366915994000000 1366915994000000
version__None None
owner__thijs thijs
cc__kkpattern
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="Infinity's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Infinity set owner to Infinity<br>Infinity set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="Infinity's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Infinity removed owner<br>Infinity set status to `new`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_5"></a>|@thijstriemstra set owner to @thijstriemstra<br>@thijstriemstra set status to `assigned`|
|-|-|

Thanks for your patch, I'll apply it.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_6"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [38260]) Apply 6403.patch: Add a missing word in the deferreds introduction document.

Author: kkpattern
Reviewer: thijs
Fixes: [#6403](https://github.com/twisted/twisted/issues/6403)
